instrumentation/github.com/go-gorm/gorm/otelgorm/callbacks.go

```diff
@@ -0,0 +1,115 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otelgorm
+
+import (
+	"fmt"
+	"strings"
+
+	"gorm.io/gorm"
+
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/label"
+	"go.opentelemetry.io/otel/semconv"
+	oteltrace "go.opentelemetry.io/otel/trace"
+)
+
+const (
+	dbTableKey     = label.Key("db.table")
+	dbCountKey     = label.Key("db.count")
```

Comment:
```suggestion
	dbRowsAffectedKey     = label.Key("db.rows_affected")
```

`db.count` seems unclear in its meaning.